ctl/install/install.go

```diff
@@ -0,0 +1,167 @@
+package install
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"k8s.io/apimachinery/pkg/api/meta"
```

Comment:
Arrange the dependency order, put the standard lib together, put Kmesh lib together and put third-party dependencies together.